CMakeLists.txt

```diff
@@ -243,7 +267,7 @@ configure_file(version.h.in ${CMAKE_CURRENT_BINARY_DIR}/version.h)
 if(BUILD_OWNCLOUD_OSX_BUNDLE)
     install(FILES sync-exclude.lst DESTINATION ${OWNCLOUD_OSX_BUNDLE}/Contents/Resources/)
     configure_file(sync-exclude.lst bin/${OWNCLOUD_OSX_BUNDLE}/Contents/Resources/sync-exclude.lst COPYONLY)
-else()
+elseif(BUILD_CLIENT)
```

Comment:
This is nonsense - don't know how it got into the diff

Reply:
@dschmidt Why is it nonsense? Shouldn't the whole preparation for sync-exclude.lst be done for BUILD_CLIENT only?